File: examples/lh-server/java/01-tasks/src/main/java/io/littlehorse/docs/tasks/Greeter.java
package io.littlehorse.docs.tasks;

import io.littlehorse.sdk.worker.LHTaskMethod;

public class Greeter {

    @LHTaskMethod("greet")
    public String greeting(String name) {
        String result = "Hello there, " + name + "!";
        System.out.println(result);
        return result;
    }
}

File: examples/lh-server/dotnet/01-tasks/Greeter.cs
using LittleHorse.Sdk.Worker;

namespace TasksExample;

public class Greeter
{
    [LHTaskMethod("greet")]
    public Task<string> Greeting(string name)
    {
        string result = $"Hello there, {name}!";
        Console.WriteLine(result);
        return Task.FromResult(result);
    }
}